### ISSUE TITLE
Amazon Polly support

### ISSUE THREAD

**Issue by mritzco** [speechmarkdown/speechmarkdown-js#62]
Hi,

I'm using Amazon Polly via API and it seems like the Alexa specification is not matching.

I'd like to contribute some fixes but would like to ask if there's any preference or coding standard for?

- How to handle different feature support for  `standard|neural`? 
- How to handle features exclusive to a few voices, like newscaster:
> Matthew or Joanna voices, which are available only in American English (en-US), Lupe, in US Spanish (es-US) and Amy, in British English (en-GB). It is only supported when using Neural format. 
- How to return `Voice` information as it's not passed as SSML but as options when calling API?

Full feature information: https://docs.aws.amazon.com/polly/latest/dg/supportedtags.html

#### Failing test on Polly:
```js
const fails = {
  "audio-standard.spec.ts": `!["https://sample-dev.s3.amazonaws.com/path/18327803923/f7fb4173-4eab-46fc-80ec-020204a615f9.mp3?AWSAccessKeyId=AKXXXXXXXXXXXXXXXXXX&Expires=1596986208&Signature=VL6q9pYc8NTjf6gKVqN0Cem0WTA="]
  Announcing Speech Markdown.`,
  "disappointed-section.spec.ts": `#[disappointed] Hey there, nice to meet you`,
  "disappointed-standard.spec.ts": `We can switch (from disappointed)[disappointed] to (really disappointed)[disappointed:"pizza"].`,
  "dj-section.spec.ts": `#[dj] Hey there, nice to meet you`,
  "emphasis-standard.spec.ts": `A (reduced)[emphasis:"reduced"] level`,
  "excited-section.spec.ts": `#[excited] Hey there, nice to meet you`,
  "excited-standard.spec.ts": `We can switch (from excited)[excited] to (really excited)[excited:"pizza"].`,
  "interjection-standard.spec.ts": `(Wow)[interjection], I didn't see that coming.`,
  "multiple-modifiers-same-text.spec.ts": `Your balance is: (12345)[number;emphasis:"strong";whisper;pitch:"high"].`,
  "pitch-standard.spec.ts": `A (high)[pitch:"high"] pitch
  A (high)[pitch:'high'] pitch`,
  "prosody-multiple-modifiers.spec.ts": `Multiple modifiers on same (text)[vol;pitch;rate]`,
  "sections-standard.spec.ts": `#[voice:'Brian'] Hey there, nice to meet you`,
  "voice-standard.spec.ts": `Why do you keep switching voices (from one)[voice:"device"] to (the other)[voice:"kendra"]?
  Why do you keep switching voices (from one)[voice:'device'] to (the other)[voice:'kendra']?`,
};
```

**Comment by johandouma** [comment]
It looks like the feature set is different too if you use Neural vs Standard - eg emphasis is an "Unsupported Neural feature". This does not seem to be mentioned anywhere on the supported tags page https://docs.aws.amazon.com/polly/latest/dg/supportedtags.html. Maybe 2 formatters are required?

**Comment by mritzco** [comment]
Yes, that's correct. support is completely different.

**Comment by arjan** [comment]
Has been fixed now that #66 is merged
